Dockerfile

```diff
@@ -24,4 +23,5 @@ COPY apache.conf /opt/docker/etc/httpd/vhost.common.d/apache.conf
 RUN echo "source ~/.additional_bashrc.sh" >> ~/.bashrc
 
 USER root
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
this leaves intermediate images behind with a lot of large garbage in them. These will be pulled

Reply:
pull updated